src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -107,7 +107,7 @@ define(function (require, exports, module) {
             // Not one of our custom events, re-trigger message event to LiveDev
             module.exports.trigger("message", [connId, msgObj.message]);
         } else if (msgObj.type === "connect") {
-            Browser.setListener();
+            //Browser.setListener();
```

Comment:
Let's revert this change please, it seems unrelated to your work.

Reply:
When a file (any type) was uploaded or deleted, console complains about the Browser.setListnere(); line, since the function was removed.
 Is that fine to leave it as it is?
<img width="310" alt="error" src="https://user-images.githubusercontent.com/22462471/35639872-8f68e85a-0689-11e8-9e0b-e29942308abb.png">

Reply:
Interesting, that makes no sense to me.  @gideonthomas do you remember us touching this at all?  We might have another bug here that needs to be fixed.

So to be clear: when you leave that line in (not commented out), and drag a file in, you get that error in the console?

Reply:
Yes, if I keep the line, I get the error every time. Then the number of the counter is increased.
However, on "Thimble", this error doesn't happen.

A friend found that the function was removed here.
https://github.com/mozilla/brackets/commit/1049e3cbbb9762b1ac1371c0744fd6c1bead56b6

Reply:
Thanks for letting us know about this, and researching the cause.  I've filed another bug to fix it: https://github.com/mozilla/thimble.mozilla.org/issues/2608.  Don't worry about it in your PR.

Reply:
Ok! Then, I'll revert the line.